lang/test/org/partiql/lang/eval/CastTestBase.kt

```diff
@@ -55,6 +42,26 @@ data class NotImplemented(val expectedQuality: CastQuality) : CastQualityStatus(
  */
 data class FixSemantics(val expectedQuality: CastQuality) : CastQualityStatus()
 
+private val typeToTypeAliases = mapOf(
```

Comment:
Rather than return a map, could this be modeled as a function that takes an `ExprValueType` and returns a list of strings? Not sure of the benefit of returning a map.

If you model it as a function, you could have a `when` expression over all the `ExprValueType`s, so if new `ExprValueType`s are added in the future, we can make sure the corresponding branch in the `when` expression is updated.

Reply:
> Could you provide some more context regarding why this change is needed and how it relates to the open type system work? I saw this PR is part of a broader issue of #772, but I'm having trouble understanding how it fits into the rest of changes.
> 
> Also, why is this change being pushed to `main` rather than to the OTS feature branch?

In OTS, we are going to enable users to define and customize type aliases for a specific type. The code here hard-codes all the type aliases, which is contradicting against what we want in OTS. 

As for why we want to merge this PR to `main`, this is because I still consider this as a refactoring to the existing code. And in general, I want to push such changes to `main` as many as possible since it reduces workload to maintain the OTS feature branch.

Reply:
I will apply this comment.